.sync.yml

```diff
@@ -18,6 +18,7 @@ Gemfile:
         version: '~>0.10.5'
     ':system_tests':
       - gem: beaker
+      - gem: beaker-rspec
```

Comment:
I'm 6 to one, half dozen to the other on this. Was more for supporting what @justinstoller might be calling in his scripts. Does it do harm if it is there?